packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -708,10 +791,14 @@ export async function getUserInfo({
 			}
 
 			const data = await response.json()
-			return UserInfoSchema.parse(data)
+			const parsedUserInfo = UserInfoSchema.parse(data)
+			log(`successfully fetched user info for user: ${parsedUserInfo.id} (${parsedUserInfo.email})`)
+			return parsedUserInfo
 		},
 	}).catch((e) => {
-		console.error('Failed to get user info', e)
+		const errorMessage = e instanceof Error ? e.message : String(e)
+		const stackTrace = e instanceof Error ? e.stack : undefined
+		log.error(`failed to get user info: ${errorMessage}`, { stackTrace })
```

Comment:
Would this be simpler and accomplish the same goal?

```suggestion
		log.error(`failed to get user info:`, e)
```